.github/workflows/Admin_Server.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Admin Panel_Server CI/CD Pipeline
+
+on:
+  push:
+    paths:
+    - 'server/**'
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'server/**'
+    branches:
+      - master
+
+env: 
+  project-directory: ./server
+
+jobs:
+  server: #Job ID Name
+    name: Build Server Project
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: npm clean install
+      run: npm ci
+      working-directory: ${{env.project-directory}}
+    - name: run ESLint
+      run: npm run lint
+      working-directory: ${{env.project-directory}}
+    - name: build server project
+      run: npm run build
+      working-directory: ${{env.project-directory}}
+    - name: run Loopback tests
```

Comment:
These tests currently require a database connection; we can omit these from the workflow for the moment.